src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -370,7 +370,53 @@ def deleteBlocks(self):
           If no => do nothing (check again next cycle)
         """
         # FIXME: figure out the proper logic for rule block deletion
+        # TODO: Update metadata after deleting block rules
```

Comment:
You want to update the block metadata once a rule deletion happens? Is it somehow a requirement?